### ISSUE TITLE
Add a section about (extra) properties

### ISSUE THREAD

**Issue by eskatos** [gradle-guides/migrating-build-logic-from-groovy-to-kotlin#18]

**Comment by keturn** [comment]
Yeah, this was a stumbling block for me. Searching led me to this: https://stackoverflow.com/questions/52401740/how-to-efficiently-populate-extra-properties-in-the-gradle-kotlin-dsl

which at this time doesn't have an answer.
